BlockchainSdk/Blockchains/Solana/SolanaSdk+.swift

```diff
@@ -203,6 +203,70 @@ extension Api {
     }
 }
 
+// MARK: swift concurrency versions of several calls
+
+extension Api {
+    func getSignatureStatuses(pubkeys: [String], configs: RequestConfiguration? = nil) async throws -> [SignatureStatus?] {
+        try await withCheckedThrowingContinuation { [weak self] continuation in
+            guard let self else {
+                continuation.resume(throwing: WalletError.empty)
+                return
+            }
+            
+            getSignatureStatuses(pubkeys: pubkeys, configs: configs) {
+                switch $0 {
+                case .failure(let error):
+                    continuation.resume(throwing: error)
+                case .success(let statuses):
+                    continuation.resume(returning: statuses)
+                }
+            }
```

Comment:
вообще я думаю можно вот так, `continuation.resume` может в себя `Result` принять

```suggestion
            getSignatureStatuses(pubkeys: pubkeys, configs: configs, completion: continuation.resume)
```

Reply:
Ну и в других местах тоже

Reply:
не совсем понял в чем смысл, swift concurrency вроде наоборот пропагандирует throwing функции вместо использования result

Reply:
не сразу понял о чем ты,  поправлю